holidays/countries/turkey.py

```diff
@@ -64,6 +64,10 @@ def _populate_public_holidays(self):
             else tr("Ulusal Egemenlik Bayramı")
         )
 
+        if self._year <= 1980:
+            # Spring Day.
+            self._add_holiday_may_1(tr("Bahar Bayramı"))
```

Comment:
I did look this up a bit and it seems like a rename of the labor day than its own thing IMO

https://www.turkishclass101.com/blog/2020/04/24/labor-day-in-turkey/